tests/dummy/app/templates/demos/sl-grid.hbs

```diff
@@ -10,13 +10,14 @@
         <h6>Template: demos/sl-grid.hbs</h6>
         <pre>\{{#sl-grid
     columns=columns
-    content=arrangedContent
+    content=model
     detailComponent="demos/sl-grid-detail"
     detailFooterComponent="demos/sl-grid-detail-footer"
     detailHeaderComponent="demos/sl-grid-detail-header"
     footerPath="demos/sl-grid/footer"
     height=400
     rowActions=rowActions
+    sendLog=(action "sendLog")
```

Comment:
Does the documentation in the demo need to explain this?  Nothing alludes to how this is all configured.